### DISCUSSION TITLE
Triangulator API

### DISCUSSION THREAD

**Discussion by ramcdona** [elalish/manifold#570] [General]
Following the discussion on the great progress on the simplified and improved Triangulator [here](https://github.com/elalish/manifold/pull/564) makes me wonder...

Is there (or will there be) a way to access the Triangulator directly?

In my program, I'm carrying another library for this purpose and it would be great to consolidate.

**Comment by elalish** [comment]
Indeed, in fact someone already asked the same thing and then contributed a PR. [doc link](https://manifoldcad.org/docs/html/group___connections.html#ga1d01ff53289da0842e539ddf8cda5c96)
